docs/network-spec/mux.tex

```diff
@@ -180,7 +180,7 @@ \section{Life Cycle of a Connection}
   1  & DeltaQ                                & Yes  & Yes \\ \hline
   2  & Chain-Sync instantiated to headers    & Yes  & No \\ \hline
   3  & Block-Fetch                           & Yes  & No  \\ \hline
-  4  & Transaction-Submission                & No   & Yes  \\ \hline
+  4  & Transaction-Submission                & Yes  & Yes  \\ \hline
```

Comment:
We have two separate `mini-protocols` for transaction submission:
* `LocalTxSubmission` - which is only used for `node-to-client`
* `TxSubmission` - only used by the `node-to-node` protocol.

They are quite a bit different, thus I suggest we have two entries in this table.

Reply:
Do these share the same protocol id?

Reply:
`TxSubmission` has id `4`
`LocaTxSubmission` has id `6`

Reply:
Fixed